reflect.go

```diff
@@ -137,6 +137,38 @@ func runInDir(program []byte, dir string) (*model.PackedPkg, error) {
 
 	return run(filepath.Join(tmpDir, progBinary))
 }
+
+func DirExists(path string) (bool, error) {
```

Comment:
Why are these utility functions in `reflect.go`?

Reply:
I'll move them elsewhere.

Reply:
Put them there because of similarity with other functions.

Reply:
I'll put them into util.go